### DISCUSSION TITLE
NORMAL support in case it exist.

### DISCUSSION THREAD

**Discussion by numerochka** [Chia-Network/chia-blockchain#11248] [Support]
Hi. I have a problem of XCH coins coming my way to my wallet for more than a month. I need a normal support, not a Keybase shit or this where you need to provide all the information of transaction seen by everyone. such a big project, does it have anything like e-mail or messengers or phone support and not tithed or Keybase sh*t to solve a problem???

**Comment by loppefaaret** [comment]
Keep a decent tone if you would be so kind!
If you need private support on keybase, cause you dont want to give possible sensitive information in an open public chat, i can fully understand that, and you are most welcome to contact @pyl on our Keybase chat. I also responded to your messages in #support . 
If you rather handle it here, on a persisting github discussion, we can do that as well - but we do not offer support through other means!

So basically, you are getting XCH send to your wallet, that you did not request ? 
If that is the case, there really isn't much we can do for you. We can't control which addresses sends transactions to which addresses, and we do not wish to do so either.

**Reply by numerochka** [reply]
I would love to keep a decent tone, but im trying to solve my problem for more than a month. My problem is that I got a chia wallet and I sent a XCH coins to my address there as I seen people say it should be synchronized in a few days or a week. but its been almost 2 months and its still not synchronized. Today in `Keybase I've been told im using "very old version" of wallet and need to upgrade (though my last upgrade was beginning of march which im surprised to be called "so old that I don't remember that version"). that was a very strange support there. I don't want to refresh wallet to not have another problems like losing wallet account where I already sent my chia coins almost 2 months ago or smith else. I just want a safe wallet where I can save my coins safely and easy to use without needing a year or so to synchronize. Now it says "NOT SYNCHRONIZED" after stopping on 1\5 of the way if I understood right. As I want my transactions to finally be complete and to see my coins in my wallet (since you provide this wallet as the wallet of your project).

**Reply by loppefaaret** [reply]
We certainly did not mean to cause you problems, and i think that it was a misunderstanding that someone suggested it was a very old client - the client have had the top menu for a long time now, and i think we might need to go all the way back to early beta's for it to not be there, if at all... I think it is just a misunderstanding of how macOS handles programs top menu's - Displaying them on the systems top bar, instead of inside the program window itself.
Since the beginning of March, we have had a few releases, and i can only recommend that if you have problems with whatever version you are using currently, that you update to the latest.
That being said, we are having some users experiencing balance displaying incorrectly, but if you are able to manually move your wallet databse out, and resync a fresh wallet database from 0, it should clear it up!
To manually move your wallet database out on macOS, you need to browse into you users home folder, and display hidden folders, then enter these directories:
`~/.chia/mainnet/wallet/db/`
Move the files in here out, while your Chia client is closed down
Once the files have been moved out, reopen your Chia client, and let it run until it displays "Synced" in green.
If you still are having issues, please let me know

**Reply by numerochka** [reply]
its too difficult for me to understand... im not a tech girl. I just wanted a wallet to have my coins there. why is it so difficult. Is that an offer (which I will try hard to understand and to do.....) to have the same wallet address where I already sent my coins even after I delete my wallet and get a newest version? maybe knowing the transaction somehow it can be possible to decline it and get my xcc coins back to address where I sent it from and forget a chia wallet as a nightmare (for me it is a nightmare, I don't need anything except wallet, I understand nothing in tech issues and all this moving databases and so on, I just want safely hold my xch in a simple wallet like bunch of other projects offer)?

**Reply by loppefaaret** [reply]
ok, perhaps this solution is something you can manage - it will reset all the configuration, and databases stored on the machine:
Open up the "Terminal" application, and issue the below command, while the Chia client is closed down:
```
rm -R ~/.chia
```
When you open up Chia again, it should only remember the key, and everything else should be as first time running it - this should also cause your wallet to rebuild a fresh wallet database from 0, and end up displaying your balance correctly
Im afraid that the only option to get the coins moved again, is to get the wallet to make another transaction back to an address that whatever service you have been using, is providing as a deposit address!
